jax/numpy/lax_numpy.py

```diff
@@ -1190,7 +1242,7 @@ def std(a, axis=None, dtype=None, out=None, ddof=0, keepdims=False):
   return sqrt(var(a, axis=axis, dtype=dtype, ddof=ddof, keepdims=keepdims))
 
 
-@_wraps(onp.ptp)
+@_wraps(onp.ptp, True)
```

Comment:
Use the kwarg here (is this `tabbed=True` or `updates=True`?)

Reply:
Done